dependency/kv_get.go

```diff
@@ -57,6 +57,8 @@ func (d *KVGetQuery) Fetch(clients *ClientSet, opts *QueryOptions) (interface{},
 		RawQuery: opts.String(),
 	})
 
+	// NOTE that the Consul HTTP KV API returns a 404 on failed gets, but the
+	// Go API ignores those for KV gets, returning nil data and error.
```

Comment:
Does the "Go API" here refer to the Consul API SDK or is it something else?

Reply:
Yeah, I mean the `github.com/hashicorp/consul/api` package. They have a separate `github.com/hashicorp/consul/sdk` package that is kind of using the SDK moniker, so I'm not 100% on what's best to call it. That was my shot. Maybe "Go API Package" would be clearer?

Reply:
Thanks for clarifying, I think that works! My only thought is maybe also adding "Consul" e.g. "Consul Go API Package"? The "something else" I was considering was the go standard library's api package (which didn't seem likely).

Reply:
Comment updated. Thanks @lornasong!